src/NHibernate/Dialect/PostgreSQL91Dialect.cs

```diff
@@ -0,0 +1,16 @@
+namespace NHibernate.Dialect
+{
+	/// <summary>
+	/// An SQL dialect for PostgreSQL 9.1 and above.
+	/// </summary>
+	/// <remarks>
+	/// PostgreSQL 9.1 disables backslash escape sequences by default.
+	/// </remarks>
+	public class PostgreSQL91Dialect : PostgreSQL83Dialect
```

Comment:
Not sure that we need to introduce new dialect. According to this: https://endoflife.date/postgresql support for Postgres 9.1 ended more than 7 years ago. I think we can enable/disable this option by default.

Reply:
So, the current state of supported PostgerSQL databases is to have backslash escapes disabled by default. I undo any PostegreSQL dialect related changes to align on this.